cli/cmd/configure_domain.go

```diff
@@ -35,7 +35,7 @@ var configureDomainParams *configureDomainCmdParams
 const installerPrefixURL = "https://raw.githubusercontent.com/keptn/keptn/"
 
 const apiVirtualServiceSuffix = "/installer/manifests/keptn/keptn-api-virtualservice.yaml"
-const apiIngressSuffix = "/installer/manifests/keptn/api-ingress.yaml"
+const apiIngressSuffix = "/installer/manifests/keptn/keptn-ingress.yaml"
```

Comment:
You also removed `api-ingress`, but I guess `keptn-ingress` is correct as this will also be the ingress for Bridge?

Reply:
Done